src/hct_mis_api/apps/payment/services/payment_gateway.py

```diff
@@ -102,13 +102,17 @@ class PaymentSerializer(ReadOnlyModelSerializer):
     remote_id = serializers.CharField(source="id")
     record_code = serializers.CharField(source="unicef_id")
     payload = serializers.SerializerMethodField()
+    extra_data = serializers.SerializerMethodField()
```

Comment:
this was removed here and now is added again?
https://github.com/unicef/hope/pull/4890/files